samples/single-page-app/config/webpack.config.js

```diff
@@ -369,23 +411,23 @@ module.exports = function(webpackEnv) {
                 customize: require.resolve(
                   'babel-preset-react-app/webpack-overrides'
                 ),
-                
-                plugins: [
-                  // This is the local reference @fullstory/babel-plugin-annotate-react
-                  // To use in your projects you would replace '../..' with '@fullstory/babel-plugin-annotate-react'
-                  '../..',
+                presets: [
                   [
-                    require.resolve('babel-plugin-named-asset-import'),
+                    require.resolve('babel-preset-react-app'),
                     {
-                      loaderMap: {
-                        svg: {
-                          ReactComponent:
-                            '@svgr/webpack?-svgo,+titleProp,+ref![path]',
-                        },
-                      },
+                      runtime: hasJsxRuntime ? 'automatic' : 'classic',
                     },
                   ],
                 ],
+                
+                plugins: [
+                   // This is the local reference @fullstory/babel-plugin-annotate-react
```

Comment:
Nit: Looks like there's an extra space before this comment line

Reply:
fixed!